be/app/Models/Follower.php

```diff
@@ -23,4 +23,9 @@ public function log()
     {
         return $this->morphOne(Activity_log::class, 'loggale');
     }
+
+    public function follows($id)
+    {
+        return $this;
+    }
```

Comment:
- what's this for? is this relationship? i don't understand :thinking:

Reply:
I am just checkint out how it will word I can remove it po